.github/workflows/assets.yml

```diff
@@ -1,26 +1,36 @@
-name: Compile assets
+name: Compile dist
 
 on:
-  push:
-    paths:
-      - 'www/assets/**'
-  pull_request:
+  pull_request_review:
+    types:
+      - submitted
     paths:
       - 'www/assets/**'
 
 jobs:
-  assets:
+  build:
+    if: github.event.review.state == 'approved'
     runs-on: ubuntu-16.04
+
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout getkirby.com
+        uses: actions/checkout@v2
         with:
           ref: ${{ github.head_ref }}
-      - uses: actions/setup-node@v2
+
+      - name: Setup node environment
+        uses: actions/setup-node@v2
         with:
           node-version: 12
```

Comment:
Do we explicitly need Node 12? We would need to update this manually when Node 12 is no longer supported. If no `node-version` is given, GitHub Actions defaults to one anyway.

Reply:
Having encountered issues with new node versions in the past, I thought it might be useful to pin the version and purposefully change update it. But we can try with default as well.